lighthouse-cli/test/smokehouse/lighthouse-runners/bundle.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
maybe this should be `launch-chrome-and-bundle.js` or something to differentiate from the pure bundle runner needed by #9873. We could bikeshed now or later